foundationdb.sln

```diff
@@ -0,0 +1,62 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Not sure if we want this SLN file since on our side we do not support MS windows -- it is supported by some other people I do not know. @ammolitor might know how to contact them.